server/controllers/userController.js

```diff
@@ -58,9 +58,12 @@ export default{
                     token: null,
                   });
             }else{
-            const token = jwt.sign({ id: result.rows.id }, 'shhhh', {
-                expiresIn: 60 * 60,
-              });
+                const payload = {
```

Comment:
when login fails we should return a text `Password or email does not match`, that way we would give a malicious user a hint.